Pod/Classes/VOKCoreDataManager.m

```diff
@@ -204,15 +210,20 @@ - (void)addPersistentStoreToCoordinator:(NSPersistentStoreCoordinator *)persiste
 }
 
 - (NSManagedObjectContext *)managedObjectContextWithConcurrencyType:(NSManagedObjectContextConcurrencyType)concurrencyType
+                                                      parentContext:(nullable NSManagedObjectContext *)parentContext
 {
-    NSPersistentStoreCoordinator *coordinator = self.persistentStoreCoordinator;
-    
-    NSAssert(coordinator, @"PersistentStoreCoordinator does not exist. This is a big problem.");
-    if (!coordinator) {
-        return nil;
-    }
     NSManagedObjectContext *context = [[NSManagedObjectContext alloc] initWithConcurrencyType:concurrencyType];
-    context.persistentStoreCoordinator = coordinator;
+    if (parentContext) {
+        context.parentContext = parentContext;
```

Comment:
[The docs](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/CoreDataFramework/Classes/NSManagedObjectContext_Class/#//apple_ref/doc/uid/TP30001182-256616) say

> As the first scenario implies, a parent context can service requests from children on different threads. You cannot, therefore, use parent contexts created with the thread confinement type

Is it worth an assertion that `parentContext.concurrencyType != NSConfinementConcurrencyType`?